util/find-doc-nits

```diff
@@ -1029,21 +1026,43 @@ my %skips = (
     'digest' => 1,
 );
 
+my %genopts; # generic options parsed from apps/include/opt.h
+
 # Check the flags of a command and see if everything is in the manpage
 sub checkflags {
     my $cmd = shift;
     my $doc = shift;
-    my %cmdopts;
+    my @cmdopts;
     my %docopts;
     my %localskips;
 
-    # Get the list of options in the command.
-    open CFH, "$openssl list --options $cmd|"
-        or die "Can list options for $cmd, $!";
+    # Get the list of options in the command source file.
+    my $src = $cmd;
+    $src = "asn1pars" if $cmd eq "asn1parse";
+    $src = "crl2p7" if $cmd eq "crl2pkcs7";
+    my $active = 0;
+    my $expect_helpstr = "";
+    open CFH, "apps/$src.c"
+        or die "Can't open apps/$src.c to list options for $cmd, $!";
     while ( <CFH> ) {
         chop;
-        s/ .$//;
-        $cmdopts{$_} = 1;
+        if ($active) {
+            last if m/^\s*};/;
+            if ($expect_helpstr ne "") {
+                next if m/^\s*#\s*if/;
+                err("$cmd does not implement help for -$expect_helpstr") unless m/^\s*"/;
+                $expect_helpstr = "";
+            } elsif (m/^\s+\{\s*"([^"]+)"\s*,\s*OPT_[A-Z0-9_]+\s*,\s*('.'|0)\s*,(.*)$/
+                       && !($cmd eq "s_client" && $1 eq "wdebug")) {
+                push @cmdopts, $1;
+                $expect_helpstr = $1;
+                $expect_helpstr = "" if $3 =~ m/^\s*"/;
+            } elsif (m/\s+(OPT_[A-Z]+_OPTIONS?)\s*(,|$)/) {
```

Comment:
The \s+ could be more forgiving.  A single white space would be sufficient, better would be white space or comma.

Five lines up the OPT pattern is `OPT_[A-Z0-9_]` why not permit digits here here too?

Reply:
Actually my idea was that the macro invocation should be at the beginning of the line (except indentation),
but this is not really needed. So relaxed the checking to require just a space or comma before `OPT_`.
Here the `[A-Z]+` is sufficient because the macro names for the generic options to not include `[0-9_]`.